packages/starknet_flutter/darwin/Classes/Managers/KeychainManager.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  KeychainManager.swift
+//  SecureEnclave
+//
+//  Created by Dimitri Dessus on 14/02/2023.
+//
+
+import Foundation
+
+// Provides a set of methods to manage encrypted data stored in the keychain.
+class KeychainManager {
+  static let starknetCipherKey = "StarknetCipher"
+  
+  // Try to save or update the encrypted cipher private key to the keychain.
+  func save(_ cipher: Data) {
+    let query = [
+      kSecValueData: cipher,
+      kSecClass: kSecClassGenericPassword,
```

Comment:
I faced some issues using ```kSecClassKey``` with ```kSecAttrAccount```  on the keychain part (works well on the [Secure Enclave](https://github.com/istornz/starknet.dart/blob/9834d73b224592e55c8f9b196eeac3cb3ab32d6f/packages/starknet_flutter/darwin/Classes/Managers/SecureEnclaveManager.swift#L33) part).

A workaround found on [StackOverflow](https://stackoverflow.com/a/58233542/5078902) indicate to use ```kSecClassGenericPassword``` if you need to use ```kSecAttrAccount```.

**Why using ```kSecAttrAccount``` ?**
We worked on a generic plugin, values like ```kSecAttrApplicationLabel``` or ```kSecAttrApplicationTag``` doesn't make sense...
In addition, ```kSecClassKey``` is like a way to save it in the keychain & retrieve it directly from the native keychain app (not necessary on our plugin, because this part is hidden from the user).

I think we can keep the ```kSecClassGenericPassword``` implementation secure on the Keychain part.

Reply:
Ok